README.md

```diff
@@ -66,6 +70,29 @@ Finally, launch the webserver and the Clojure REPL:
 
 and open [http://localhost:1042/](http://localhost:1042/).
 
+### Using Docker
+
+*See [this issue](https://github.com/mtgred/netrunner/issues/6397).*
+
+You'll need to install [Docker](https://docs.docker.com/get-docker/) and [Docker-Compose](https://docs.docker.com/compose/install/). After that, just run `$ docker-compose up` in the project directory (or do the GUI-equivalent of this). If this fails because it "couldn't fetch dependencies", try again, it was just a networking error.
```

Comment:
Small note on Docker Compose: you can simply run `docker compose ...` on recent versions of Docker ;)